doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,27 +57,31 @@ Values are provided in kilobytes (KB).
       The following table lists memory requirements for samples running on the :ref:`nRF52840 DK <programming_board_names>` (:ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`).
 
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                           |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                           |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

Comment:
Argh, this should be actually `MCUboot` :/ Can you change this here and elsewhere on this page if you happen to make pushes?